itests/hive-unit/src/test/java/org/apache/hive/service/cli/thrift/TestThriftHttpCLIServiceFeatures.java

```diff
@@ -138,16 +138,15 @@ public boolean hasCookieHeader() {
   @BeforeClass
   public static void setUpBeforeClass() throws Exception {
     // Set up the base class
-    ThriftCLIServiceTest.setUpBeforeClass();
+    ThriftCLITestUtils.setUpBeforeClass();
```

Comment:
doesn't parent method gets invoked before that?

Reply:
For this particular class, we will have to do an explicit call to initialise hs2, and other members as the current setUpBeforeClass() method overrides it's parent method.

Reply:
it's static, it can't override the parent

Reply:
see how same is done in `ITestDbTxnManager`.setupDb that extends `DbTxnManagerEndToEndTestBase`